### PR TITLE
Feature/451998 grant registration confirmation - Build 'Are you sure you want to grant this registration? page

### PR DESCRIPTION
**Pull request by karthikeviden**
closing this PR due to more commits being shown. This work is now in the new PR - https://github.com/DEFRA/epr-regulator-service/pull/103

### CODE REVIEW

File: src/EPR.RegulatorService.Frontend.Core/Services/IFacadeService.cs [hunk 1]
```diff
@@ -55,4 +55,6 @@ Task<PaginatedList<T>> GetOrganisationSubmissions<T>(
 
     Task<PaginatedList<RegistrationSubmissionOrganisationDetails>> GetRegistrationSubmissions(RegistrationSubmissionsFilterModel filters);
     RegistrationSubmissionOrganisationDetails GetRegistrationSubmissionDetails(Guid organisationId);
+
+    Task<EndpointResponseStatus> SubmitRegulatorDecisionAsync(RegulatorDecisionRequest request);
```
Comment (karthikeviden) [inline src/EPR.RegulatorService.Frontend.Core/Services/IFacadeService.cs:59]: Revisit the name

File: src/EPR.RegulatorService.Frontend.Core/Services/FacadeService.cs [hunk 2]
```diff
@@ -357,4 +357,6 @@ public RegistrationSubmissionOrganisationDetails GetRegistrationSubmissionDetail
 
         return null;
     }
+
+    public async Task<EndpointResponseStatus> SubmitRegulatorDecisionAsync(RegulatorDecisionRequest request) => await Task.FromResult(EndpointResponseStatus.Success);
```
Comment (karthikeviden) [inline src/EPR.RegulatorService.Frontend.Core/Services/FacadeService.cs:361]: Revisit the implementation

File: src/EPR.RegulatorService.Frontend.Core/Services/MockedFacadeService.cs [hunk 3]
```diff
@@ -406,4 +406,6 @@ public RegistrationSubmissionOrganisationDetails GetRegistrationSubmissionDetail
 
         return objRet;
     }
+
+    public async Task<EndpointResponseStatus> SubmitRegulatorDecisionAsync(RegulatorDecisionRequest request) => await Task.FromResult(EndpointResponseStatus.Success);
```
Comment (karthikeviden) [inline src/EPR.RegulatorService.Frontend.Core/Services/MockedFacadeService.cs:410]: Revisit the implementation

File: src/EPR.RegulatorService.Frontend.Core/Models/RegistrationSubmissions/RegulatorDecisionRequest.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+namespace EPR.RegulatorService.Frontend.Core.Models.RegistrationSubmissions;
+
+using EPR.RegulatorService.Frontend.Core.Enums;
+
+public class RegulatorDecisionRequest
```
Comment (JulianYoungEviden) [inline src/EPR.RegulatorService.Frontend.Core/Models/RegistrationSubmissions/RegulatorDecisionRequest.cs:5]: Can you rename this to RegulatorRegistrationDecisionRequest please?

Comment (karthikeviden) [reply src/EPR.RegulatorService.Frontend.Core/Models/RegistrationSubmissions/RegulatorDecisionRequest.cs:5]: yes, sure I am working on these. Thanks for the name

File: src/EPR.RegulatorService.Frontend.Core/Services/IFacadeService.cs (hunk 1 above)
Comment (karthikeviden) [reply src/EPR.RegulatorService.Frontend.Core/Services/IFacadeService.cs]: done

File: src/EPR.RegulatorService.Frontend.Core/Services/MockedFacadeService.cs (hunk 3 above)
Comment (karthikeviden) [reply src/EPR.RegulatorService.Frontend.Core/Services/MockedFacadeService.cs]: done

File: src/EPR.RegulatorService.Frontend.Core/Services/FacadeService.cs (hunk 2 above)
Comment (karthikeviden) [reply src/EPR.RegulatorService.Frontend.Core/Services/FacadeService.cs]: done, as discussed with @JulianYoungEviden

File: src/EPR.RegulatorService.Frontend.Core/Models/RegistrationSubmissions/RegulatorDecisionRequest.cs (hunk 4 above)
Comment (karthikeviden) [reply src/EPR.RegulatorService.Frontend.Core/Models/RegistrationSubmissions/RegulatorDecisionRequest.cs]: done, as discussed with @JulianYoungEviden

**Pull request closed without merging**